README.md

```diff
@@ -4,15 +4,47 @@ These scripts are run at https://ci.adoptopenjdk.net/view/git-mirrors/ and are r
 
 ## For developers
 
-OpenJDK Source Control repositories are managed using Mercurial. The scripts within this project mirror those Mercurial repositories
-to github.com/adoptium/jdk\<NN\>u , which then serves as a mirror for Adoptium to build the Hotspot variants, as well as providing branches for adding any Adoptium vendor "patches" needed.
+OpenJDK Source Control repositories are now managed using the GitHub (project Skara). The scripts within the projects that use Mercurial repositories
+from github.com/adoptium/jdk\<NN\>u , would not be invalidated but would be converted to Git. The existing master Mercurial repos will at least be kept as read-only archives for a defined transition period. Longer term, a Mercurial URL to Git URL translator might be put into place.
 
-The OpenJDK Mercurial repositories fall under three categories:
-```
-- JDK8 Update Repositories   : hg.openjdk.java.net/jdk8u/jdk8u/(corba|hotspot|jaxp|jaxws|nashorn|jdk|langtools)
-- JDK11+ Update Repositories : hg.openjdk.java.net/jdk-updates/jdk<NN>u
-- JDK "next" Repository      : hg.openjdk.java.net/jdk/jdk
-```
+There is a program put in place to convert OpenJDK Mercurial repositories to a Git repository. It uses the git-fast-import protocol to import Mercurial changesets into Git, and it adjusts existing commit messages to align with Git best practices. A commit message for the Mercurial jdk/jdk repository has this structure:
```

Comment:
I'm not sure we need all of this detail.  We simply mirror the Git repos at OpenJDK and don't have to worry about the mecurial -> git conversion side of things.

Reply:
Alright then, I would resolve this ASAP. Thank you @karianna